### DISCUSSION TITLE
Floating in the Crash Reports

### DISCUSSION THREAD

**Discussion by Raytwo** [Raytwo/ARCropolis#480] [Announcements]
### New feature(s)
* Reworked auto-update prompt to show the changes before accepting an update (@Coolsonickirby)

### Improvement(s)
* Improved navigation and responsiveness in all the ARCropolis menus (@Coolsonickirby)
* Tweaked the message about conflicts.json to be clearer

### Bug fix(es)
* Creating and changing workspaces won't crash the game on boot anymore (@Raytwo)

### Note
* If you go online with added slots, you take full responsibility on whatever happens to your Switch or Nintendo Online account.
* We do not endorse going online with added slots and we are only trying to make it more safe to not risk running into trouble with Nintendo. If you get banned, that's on you. If you get disconnected, that's also on you. Choose carefully.

# Read this
Skyline is not shipped with ARCropolis anymore.
Please read [this issue](https://github.com/Raytwo/ARCropolis/issues/157) to know how to proceed from now on.

<hr /><em>This discussion was created from the release <a href='https://github.com/Raytwo/ARCropolis/releases/tag/v3.12.0'>Floating in the Crash Reports</a>.</em>

**Comment by ErickGarzaL** [comment]
Hi, good, I just wanted to ask what happened to the ultimate folder? How do I install the mods?
